test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs

```diff
@@ -14,6 +14,8 @@ namespace Microsoft.AspNetCore.Localization.FunctionalTests
 {
     public class TestRunner
     {
+        private const string ApplicationBasePath = "http://localhost:0";
```

Comment:
It also looks like this code was copied from another repo and that's not a good solution for something that's going to live this long.